frontend/src/pages/analytics/dashboard/create/CreateDashboard.tsx

```diff
@@ -0,0 +1,263 @@
+              errorText={
+                dashboardNameRequiredError
+                  ? t('analytics:valid.dashboardNameEmptyError')
+                  : ''
+              }
+            >
+              <SpaceBetween direction="vertical" size="s">
+                <Input
+                  placeholder={
+                    t('analytics:dashboard.createInputNamePlaceholder') || ''
+                  }
+                  value={curDashboard.name ?? ''}
+                  onChange={(e) => {
+                    setDashboardNameRequiredError(false);
+                    setCurDashboard((prev) => {
+                      return {
+                        ...prev,
+                        name: e.detail.value,
+                      };
+                    });
+                  }}
+                />
+              </SpaceBetween>
+            </FormField>
+            <div className="mt-10">
+              <FormField label={t('analytics:dashboard.createDesc')}>
+                <Textarea
+                  placeholder={
+                    t('analytics:dashboard.createDescPlaceholder') || ''
+                  }
+                  rows={3}
+                  value={curDashboard.description}
+                  onChange={(e) => {
+                    if (
+                      new RegExp(XSS_PATTERN).test(e.detail.value) ||
+                      e.detail.value.length > MAX_USER_INPUT_LENGTH
+                    ) {
+                      return false;
+                    }
+                    setCurDashboard((prev) => {
+                      return { ...prev, description: e.detail.value };
+                    });
+                  }}
+                />
+              </FormField>
+            </div>
+
+            <FormField
+              label={t('analytics:dashboard.createSheets')}
+              errorText={
+                dashboardSheetTooMuchError
+                  ? t('analytics:valid.dashboardSheetTooMuchError')
+                  : ''
+              }
+            >
+              <ColumnLayout columns={2} variant="text-grid">
+                <Input
+                  onChange={({ detail }) => setSheetName(detail.value)}
+                  value={sheetName}
+                />
+                <Button
+                  iconName="add-plus"
+                  onClick={() => {
+                    if (!sheetName.trim()) {
+                      return false;
+                    }
+                    if (sheetNames.length >= 10) {
+                      setDashboardSheetTooMuchError(true);
+                      return false;
+                    }
+                    setSheetNames(sheetNames.concat({ label: sheetName }));
+                  }}
+                />
+              </ColumnLayout>
+            </FormField>
+            <TokenGroup
+              onDismiss={({ detail: { itemIndex } }) => {
+                setSheetNames([
+                  ...sheetNames.slice(0, itemIndex),
```

Comment:
Need make sure sheetNames length large than 2?

Reply:
This is safe:
```
If end < -array.length, 0 is used.
If end >= array.length or end is omitted, array.length is used, causing all elements until the end to be extracted.
If end is positioned before or at start after normalization, nothing is extracted.
```